src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,191 @@
+                    int index = line.indexOf(" ");
+                    int pos = line.indexOf("/");
+                    String activity = line.substring(index, pos - 1);
+                    String dueDate = line.substring(pos + 3, line.length());
+                    Deadline deadline = new Deadline(activity, dueDate);
+                    tasks.add(deadline);
+                    System.out.println("Added: " + "\n" + deadline.toString() + "\n");
+                    System.out.println("You have " + tasks.size() + " number of tasks in you list.");
+                    System.out.println("____________________________________________________________\n");
+                    appendToFile(f.getAbsolutePath(), deadline.toString() + "\n");
+                }
+                else{
+                    System.out.println("🤪Drink coffee and enter a valid deadline command");
+                    System.out.println("____________________________________________________________\n");
+                }
+            }
+
+            else if (line.contains("event")) {
+                if (line.length() > 6 && line.contains("/")){
+                    int index = line.indexOf(" ");
+                    int startDate = line.indexOf("/");
+                    String activity = line.substring(index, startDate);
+                    String timeFrame = line.substring(startDate + 3, line.length());
+                    Event event = new Event(activity, timeFrame);
+                    tasks.add(event);
+                    System.out.println("Added: " + "\n" + event.toString() + "\n");
+                    System.out.println("You have " + tasks.size() + " number of tasks in you list.");
+                    System.out.println("____________________________________________________________\n");
+                    appendToFile(f.getAbsolutePath(), event.toString() + "\n");
+                }
+                else{
+                    System.out.println("🤪Drink coffee and enter a valid event command");
+                    System.out.println("____________________________________________________________\n");
+                }
+            }
+
+            else if(tasks.isEmpty()){
+                System.out.println("🤪The list is empty. Do something in order to have tasks in your list");
+                System.out.println("____________________________________________________________\n");
+            }
+
+            else if (!line.equals("list")){
+                System.out.println("🤪Sorry enter a valid Duke command");
+                System.out.println("____________________________________________________________\n");
+            }
+            line = in.nextLine();
+        }
+        System.out.println("____________________________________________________________\n");
+        System.out.println(" Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________\n");
+    }
+}
```

Comment:
this method can be significantly improved by following better SLAP.
see what all operations can be extracted as separate methods.